template/{% if test %}ui-tests{% endif %}/tests/{{python_name}}.spec.ts.jinja

```diff
@@ -32,6 +32,9 @@ test('should emit an activation console message', async ({ page }) => {
 
   await page.getByRole('button', { name: 'Rename' }).click();
 
+  // Close file opened as editor
+  await page.activity.closePanel('test.my_type');
```

Comment:
@jtpio @krassowski I would like your input on this.

With 4.0.5, the `page.filebrowser.open(filename)` code now fails if a file is opened multiple times (different viewers) as in this case. My question, should it be fixed per test as here or fix upstream by allowing to find any number of panel opened for a given filename?

Reply:
> (different viewers) as in this case. My question, should it be fixed per test as here or fix upstream by allowing to find any number of panel opened for a given filename?

I would say upstream ideally? Could this also fix https://github.com/jupyterlab/jupyterlab/issues/13188 maybe?

Reply:
I opened https://github.com/jupyterlab/jupyterlab/pull/14983